compiler/rustc_hir/src/hir.rs

```diff
@@ -2345,7 +2345,8 @@ impl ImplItemId {
     }
 }
 
-/// Represents anything within an `impl` block.
+/// Represents an associated item within an impl block.
+/// Refer to [`Impl`] for an impl block declaration.
```

Comment:
```suggestion
/// Represents an associated item within an impl block.
///
/// Refer to [`Impl`] for an impl block declaration.
```

Putting the first sentence into its own paragraph tells rustdoc (https://doc.rust-lang.org/nightly/nightly-rustc) that only it should be part of the item summary as shown on module overview pages.

Reply:
Thanks!